applications/GeoMechanicsApplication/tests/test_thermal_element/test_thermal_heat_flux_2D10N.gid/MaterialParameters.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Should we retain the .gid in the directory names for the tests? We are only having the generated .json and .mdpa files here. What created those is no longer important, the files necessary for GiD are not here.

Reply:
removed .gid extenstion for the folders' names